main.go

```diff
@@ -59,6 +59,7 @@ type Config struct {
 	SyslogAddr          string   `envconfig:"SYSLOG_ADDR" default:"127.0.0.1:514"`
 	ContainerLogsStdout bool     `envconfig:"CONTAINER_LOGS_STDOUT" default:"false"`
 	SendDockerLabels    []string `envconfig:"SEND_DOCKER_LABELS" default:""`
+	LogHostname         string   `envconfig:"LOG_HOSTNAME"` // Name we log as
```

Comment:
"Name we log as"?

Reply:
Explained in more detail in the README

Reply:
Ah, I'm asleep and couldn't parse it in my head :) Please ignore.

Reply:
Haha no worries :) Thanks for reviewing!